### PR TITLE
First pass at mirror operation and command

### LINKED ISSUE #144: Add a mirror command to cache remote ontology imports closure on local disk
Similar to https://github.com/owlcollab/owltools/wiki/Import-Chain-Mirroring

### PR DESCRIPTION
**Pull request by cmungall**
fixes #144

Adds a command `mirror`

TODO: add to examples/README

### CODE REVIEW

File: robot-core/src/test/resources/import_test.owl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<rdf:RDF
+     xmlns:dc="http://purl.org/dc/elements/1.1/"
+     xmlns:foaf="http://xmlns.com/foaf/0.1/"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+  >
+  <owl:Ontology rdf:about="https://github.com/ontodev/robot/robot-core/src/test/resources/import_test.owl">
+    <owl:imports rdf:resource="https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl"/>
```
Comment (jamesaoverton) [inline robot-core/src/test/resources/import_test.owl]: This IRI doesn't point to a file.

**Comment by cmungall** [comment]
it's intentional - it's mapped by the catalog. It's intended to simulate an external ontology import without introducing a network dependency into the test

**Comment by jamesaoverton** [comment]
Hmm. I haven't used the GitHub review stuff before. You should just merge this when you're ready. If I accidentally blocked something, let me know and I'll fix it.

**Comment by cmungall** [comment]
The review feature is cool. But as an admin of this repo I can dismiss the review 😃

**Pull request merged**